tx/textual/valuerenderer/coins.go

```diff
@@ -76,7 +83,120 @@ func (vr coinsValueRenderer) Format(ctx context.Context, v protoreflect.Value) (
 	}
 }
 
-func (vr coinsValueRenderer) Parse(_ context.Context, screens []Screen) (protoreflect.Value, error) {
-	// ref: https://github.com/cosmos/cosmos-sdk/issues/13153
-	panic("implement me, see #13153")
+func (vr coinsValueRenderer) Parse(ctx context.Context, screens []Screen) (protoreflect.Value, error) {
+	if len(screens) != 1 {
+		return nilValue, fmt.Errorf("expected single screen: %v", screens)
+	}
+
+	if screens[0].Text == emptyCoins {
+		if vr.fd != nil && vr.fd.IsList() {
+			return protoreflect.ValueOfList(listpb.NewGenericList([]*basev1beta1.Coin{})), nil
+		}
+
+		return protoreflect.ValueOfMessage((&basev1beta1.Coin{}).ProtoReflect()), nil
+	}
+
+	coins := strings.Split(screens[0].Text, ", ")
```

Comment:
What happens if the format splitter is "," and not ", "? I'd suggest perhaps we play it safe and use "," then strings.TrimSpace each value before parsing it.